src/modules/control_allocator/ActuatorEffectiveness/ActuatorEffectivenessStandardVTOL.cpp

```diff
@@ -94,13 +116,13 @@ void ActuatorEffectivenessStandardVTOL::setFlightPhase(const FlightPhase &flight
 	// update stopped motors
 	switch (flight_phase) {
 	case FlightPhase::FORWARD_FLIGHT:
-		_stopped_motors = _mc_motors_mask;
+		_stopped_motors |= _mc_motors_mask;
 		break;
 
 	case FlightPhase::HOVER_FLIGHT:
 	case FlightPhase::TRANSITION_FF_TO_HF:
 	case FlightPhase::TRANSITION_HF_TO_FF:
-		_stopped_motors = 0;
+		_stopped_motors &= ~_mc_motors_mask;
```

Comment:
Would this stop the pusher in hover? It's needed for pusher support.

Reply:
> Would this stop the pusher in hover? It's needed for pusher support.

No, exactly. That's what I took care of. It only enables and disables mc motors but the pusher is only off when it's commanded zero thrust. This can be extended to where the controller decides to always run it even if just idling but currently for 1.14, it just works again like it was.